### PR TITLE
Updated rofi config syntax. Added theme for Awesome window manager.

### PR DESCRIPTION
**Pull request by wos-dv**

### CODE REVIEW

**Comment by roosta** [comment]
Hi, thanks for your contribution!

In the rofi readme there are trailing whitespace that should be addressed. The commits "Improved readme.md." should be squashed. The commit messages should be in present tense. (Improve readme.md), and remove trailing period in commit messages.

I'm unable to test awesome theme easily, hopefully there are someone in the community able to test it. Maybe add a screenshot somewhere?

File: rofi/README.md [hunk 1]
```diff
@@ -1,19 +1,34 @@
...
+Rofi theme created to work with [Srcery colors](https://github.com/srcery-colors/srcery-vim). To use it include this in your rofi config (outside the configuration block, at the end of the config.rasi file):
 ```conf
-rofi.theme: ~/path/to/srcery.rasi
+@theme "~/path/to/srcery.rasi"
+```
+or set the theme property inside the configuration block:
+```conf
+configuration {
+  /* Other properties omitted. */
+
```
Comment (roosta) [inline rofi/README.md:11]: trailing whitespace, indentation issues

**Review by roosta — CHANGES_REQUESTED** [review]

**Comment by wos-dv** [comment]
Reorganized the committs. Added a screenshot for AwesomeWM. I would fix the indentation too, but I don't know what is wrong with it. Can you elaborate?

**Comment by roosta** [comment]
image
Just some trailing whitespace, dunno why the indentation was wrong previously, thought maybe you used a tab char or something, but seems to be ok now. Love the new log, much cleaner :+1:

**Comment by wos-dv** [comment]
Removed the spaces.

**Review by roosta — APPROVED** [review]

**Comment by MindTooth** [comment]
Thank you for the contrib. 👍🏻

**Pull request merged**